test/csharp/test_PSConfiguration.cs

```diff
@@ -33,6 +33,12 @@ public PowerShellPolicyFixture()
             systemWideConfigDirectory = Utils.DefaultPowerShellAppBase;
             currentUserConfigDirectory = Utils.GetUserConfigurationDirectory();
 
+            if (!Directory.Exists(currentUserConfigDirectory))
+            {
+                // Create the CurrentUser config directory if it doesn't exist
+                Directory.CreateDirectory(currentUserConfigDirectory);
+            }
+
             systemWideConfigFile = Path.Combine(systemWideConfigDirectory, "PowerShellProperties.json");
             currentUserConfigFile = Path.Combine(currentUserConfigDirectory, "PowerShellProperties.json");
```

Comment:
I believe `Properties` should be replaced with `Config`.
`PowerShellConfig.json` or `PowerShell.Config.json` or `PowerShell.config.json`

Reply:
I don't like this file name either. I will see if I can rename it.

Reply:
Pushed new commit to rename the file. The file name is now `powershell.config.json`.